swagger/swagger-generator/generator-core/pom.xml

```diff
@@ -72,7 +72,6 @@
     <dependency>
       <groupId>javax.xml.bind</groupId>
       <artifactId>jaxb-api</artifactId>
-      <scope>test</scope>
```

Comment:
Yes, that's the way to fix it. But should we put it below these test dependencies?

Reply:
yes, It is better to adjust the order of dependencies